src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -884,7 +884,77 @@ protected void ClearPreparing()
         {
             Interlocked.Exchange(ref _isPreparing, 0);
         }
-
         #endregion
+
+        /// <summary>
+        /// Enable to decode an audio frame for exporting PCM from a data.
+        /// </summary>
+        /// <remarks><para>This function must be called before calling player_prepare() or player_prepare_async().
+        /// A registered callback is called in a separate thread(not in the main loop).
```

Comment:
Do user know this? User doesn't care about this, they just want to get MediaPacket.
If you want to give attention about something caused by this(e.g. do not process UI job in event handler, ...), 
how about describe it directly?